build/main_test.go

```diff
@@ -34,3 +34,45 @@ func ExampleTransactionBuilder() {
 	fmt.Printf("tx base64: %s", txeB64)
 	// Output: tx base64: AAAAADZY/nWY0gx6beMpf4S8Ur0qHsjA8fbFtBzBx1cbQzHwAAAAZAAAAAAAAAABAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAALSRpLtCLv2eboZlEiHDSGR6Hb+zZL92fbSdNpObeE0EAAAAAAAAAAB3NZQAAAAAAAAAAARtDMfAAAABA2oIeQxoJl53RMRWFeLB865zcky39f2gf2PmUubCuJYccEePRSrTC8QQrMOgGwD8a6oe8dgltvezdDsmmXBPyBw==
 }
+
+// ExamplePathPayment creates and signs a simple transaction with PathPayment operation, and then
+// encodes it into a base64 string capable of being submitted to stellar-core.
+func ExamplePathPayment() {
```

Comment:
Notice that the Path builder object can have chained calls to build the path.  One alternative formulation, provided we went with the `NativeAsset` change proposed in the comment above:  

```
Path(NativeAsset{}, CreditAsset{"USD", "Blah"})
```